source/views/building-hours/report/overview.js

```diff
@@ -203,13 +181,11 @@ export class BuildingHoursProblemReportView extends React.PureComponent<
 
 		return (
 			<ScrollView>
-				<View style={styles.helpWrapper}>
-					<Text style={styles.helpTitle}>Thanks for spotting a problem!</Text>
-					<Text style={styles.helpDescription}>
-						If you could tell us what the new times are, we&rsquo;d greatly
-						appreciate it.
-					</Text>
-				</View>
+				<InfoHeader
+					message="If you could tell us what the new times are, we&rsquo;d greatly
+                        appreciate it."
```

Comment:
Why is this on a newline?

Reply:
I have changed this under the assumption that this was not intentional.